exercises/takudzwa_makoni/ex01_basics/src/exercise01.cpp

```diff
@@ -0,0 +1,100 @@
+
+#include "pch.h"
+#include <iostream>
+#include <fstream>
+#include <list>
+#include <iomanip>
+#include <string>
+#include <algorithm>
+#include <cctype>
+#include <iomanip>
+
+
+using namespace std;
+list<string> searchList = {"which","holmes","there","could","photograph"}; // list of words to be searched.
```

Comment:
We'd generally avoid using `std::list` (implemented as a linked list) unless there is a very good reason to.

As you have a fixed number of items here `std::array` would be a good choice, otherwise `std::vector`.

Also as this is not intended to be modified this should be declared `const`.

Reply:
converted to std::vector